airbyte-integrations/connectors/destination-duckdb/Dockerfile

```diff
@@ -1,42 +1,34 @@
-FROM python:3.9.11 as base
+FROM python:3.9.11
 # FROM python:3.9.11-alpine3.15 as base
 # switched from alpine as there were tons of errors (in case you want to switch back to alpine)
 # - https://stackoverflow.com/a/57485724/5246670
 # - numpy error: https://stackoverflow.com/a/22411624/5246670
 # - libstdc++ https://github.com/amancevice/docker-pandas/issues/12#issuecomment-717215043
 # - musl-dev linux-headers g++ because of: https://stackoverflow.com/a/40407099/5246670
 
-# build and load all requirements
-FROM base as builder
 WORKDIR /airbyte/integration_code
 
-# upgrade pip to the latest version
+# Upgrade system packages and install Poetry
 RUN apt-get update && apt-get -y upgrade \
-    && pip install --upgrade pip
+    && pip install --upgrade pip \
+    && pip install poetry
```

Comment:
@bnchrch I think we should add `poetry` to our base python image. Wdyt? I can add it in https://github.com/airbytehq/airbyte/pull/30303